python/tvm/relax/frontend/onnx_frontend.py

```diff
@@ -325,22 +329,24 @@ def _impl_v16(cls, bb, inputs, attr):
 
 def _get_convert_map(opset):
     return {
-        "MatMul": MatMul.get_converter(opset),
-        "Concat": Concat.get_converter(opset),
-        "Add": Add.get_converter(opset),
-        "Mul": Mul.get_converter(opset),
-        "Cast": Cast.get_converter(opset),
-        "Gather": Gather.get_converter(opset),
-        "Gemm": Gemm.get_converter(opset),
-        "Reshape": Reshape.get_converter(opset),
-        "Div": Div.get_converter(opset),
-        "Sigmoid": Sigmoid.get_converter(opset),
-        "Softmax": Softmax.get_converter(opset),
-        "Transpose": Transpose.get_converter(opset),
-        "Unsqueeze": Unsqueeze.get_converter(opset),
-        "Gelu": Gelu.get_converter(opset),
-        "BiasGelu": BiasGelu.get_converter(opset),
-        "Where": Where.get_converter(opset),
+        "MatMul": tvm.relay.frontend.onnx.MatMul,
```

Comment:
stylistic preference, but I'd rather use `relay.frontend.onnx.MatMul` (i.e. drop the tvm)